### PR TITLE
Improve data entry ui

### LINKED ISSUE #169: Editing Chronicle: When adding new STs, the system re-adds everyone checked but the original ST
The Edit Chronicle page has a checkbox format for adding STs. The Owner of the Chronicle has a greyed out checkbox, and it is possible to add new STs by checking a box next to their name and clicking "submit."

When this is done the first time, their name is added once to the list of STs. On further times (e.g., to add more STs), their names are added again, leading to repetition of the names.

### LINKED ISSUE #253: Add active/inactive to relevant models
- Merits
- Disciplines

### LINKED ISSUE #255: Create UI for data entry for STs
Merits, questionnaires, powers

### PR DESCRIPTION
**Pull request by cmho**
Closes #255.
Closes #253.
Closes #169.

this got... to be a lot, also there's a LOT of database schema changes. there will maybe need to be some database editing once this goes live to ensure stuff gets hooked back together, but also that can now be done in the admin UI now :v

### CODE REVIEW

File: app/views/admin/index.slim [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.row.character_index_page
+	.small-12.small-centered.columns
+
+		h1 Admin
```
Comment (stephmarx) [inline app/views/admin/index.slim]: The `h1` tag should only be used for the main header of the site. This should therefore be an `h2`

Comment (sublimemarch) [inline app/views/admin/index.slim]: The `h1` tag should only be used for the main header of the site. This should therefore be an `h2`

**Review by sublimemarch — CHANGES_REQUESTED** [review]
I can't get this code to run and get this error when I `be rake db:reset`

**Comment by sublimemarch** [comment]
Error I get when I try to run this code.

screen shot 2016-09-18 at 7 55 45 pm

**Comment by sublimemarch** [comment]
- Create documentation on how making new powers and power groups works. Consider adding help text.
- Add help text for "Allowed Ratings" in merits. Should it be `[a, b, c]`?
- When making a new flaw, I can't select the character type dropdown.
- Add help text to new character type form. Perhaps examples like `Power Stat (example: Blood Potency)`
- Error when loading character/new:

screen shot 2016-09-18 at 8 31 26 pm

**Comment by sublimemarch** [comment]
Feedback for help text:

On `/admin` page:
- Rename "Character Types" link to "Creature Types." In general, this is what we should display to the user.
- The example of "Circle of the Crone Rituals" in "Power Types" is incorrect. They are a power group within the power type "Blood Sorcery". (according to Mark)

On `/power_types`:
- What does a status of "true" mean?
- There's another "Create New" button at the bottom of the page.
- On the new/edit form, change the "Character Type" select from a multi-select box to check boxes.

On `/power_groups`:
- On the new/edit form, what does "Active?" mean?
- On the new/edit form, change the select form to radio buttons. Should these also be alphabetical for ease of use? You should only be able to select one power type.

On `/powers`:
- On the new/edit form, change the select form to radio buttons. Make these alphabetical, and ideally give them unselectable headings of power type (it looks like they're grouped that way right now). You should only be able to select one power group.

On `/merits`:
- Add helper text explaining what specification and description mean.
- Should users be able to create new merit categories?

On `/flaws`:
- On new/edit form, why are there both a "Description" box and a "requires description" checkbox?

In general, we should change "true/false" in the active/specification/description column to "yes/no" so humans who aren't us can read it easier.

**Review by sublimemarch — APPROVED** [review]

**Pull request merged**